store/helpers/pages/page.js

```diff
@@ -11,7 +11,7 @@ class Page {
   }
 
   get isLast() {
-    return this.data.length < this.maxSize
+    return this.data.length === 0
```

Comment:
How is this affected?

Reply:
You can find more in the task description. Basically when 100 records are requested API can return only 10 but it doesn't mean there are no more records. The last page is detected when empty array is sent.

Reply:
Yes but this is not correct behaviour, is it? @valerapro could you please look at this?

Reply:
It was fixed like that. I just did the frontend part.  Like I don't like how it works but I understand it's hard to process it on the backend. So two lines of changes on the frontend are sufficient for me.